test/unit/parser.test.js

```diff
@@ -192,7 +192,9 @@ test('includeFile replaces <include src="include.md#doesNotExist"> with error <d
   const baseUrlMap = {};
   baseUrlMap[ROOT_PATH] = true;
 
-  const markbinder = new MarkBind();
+  const markbinder = new MarkBind({
+    errorHandler: () => {},
```

Comment:
Since we're constructing and testing the error message in `expected`, how about extracting that and `expect(e.message).toEqual` that here?

```diff
+ const expectedErrorMessage = `No such segment 'doesNotExist' in file: ${includePath}`
+   + `\nMissing reference in ${indexPath}`;

const markbinder = new MarkBind({
+ errorHandler: (e) => { expect(e.message).toEqual(expectedErrorMessage); },
});

// ...

const expected = [
  '# Index',
- `<div style="color: red">No such segment 'doesNotExist' in file: ${includePath}`,
- `Missing reference in ${indexPath}</div>`,
+ `<div style="color: red">${expectedErrorMessage}</div>`,
  '',
].join('\n');
```